### ISSUE TITLE
Add generate random characters

### ISSUE THREAD

**Issue by yordis** [igas/faker#84]
I normally use this
```elixir
def get_random_string(length \\ 5) do
    :crypto.strong_rand_bytes(length)
    |> Base.encode64
  end
```
Is there any way to do this right now?
If no, should I create a PR for add it?

**Comment by igas** [comment]
Take a look at `Faker.format/1` is it what you are looking for?

**Comment by yordis** [comment]
@igas no really, the my example just generate a random string with length of `length`

**Comment by tobyhinloopen** [comment]
I had to use my own methods as well for generating random digits & letters. Would be nice if there were some utilities in Faker to generate random numbers, random lowercase characters, random anycase characters and random alphanumeric characters, or generate X random things based on a list of elements.

See: https://github.com/igas/faker/blob/master/lib/faker/code/iban.ex#L108-L114

**Comment by igas** [comment]
I agree with @tobyhinloopen, I see faker as random but real life data, like name, places etc. Just random generators - not sure. On the other hand if ppl expected this from the lib, why not to bring it in core. We can extract it in the future in `faker-utils` as a deps if we want it.

@tobyhinloopen if you have time please send PR to start discussion.

**Comment by yordis** [comment]
@igas please don't create another package when this package can have that code without any problem. At the end of the day this package is mean to be use for generate fake data, and is not fro front end where the size of the files matters. Please I beg you dont create another package.

What is the problem with generate random string? From my issue that's what I need and probably some people will have it. I don't want to be dealing with putting some `format` for generate  the string, isn't my use case.

I have no problem keeping the function in my code base but I don't get what is the problem to have something like that?

Let say I want to use for generate `random password` which is just this function ...

**Comment by yordis** [comment]
@igas I tried to use `Faker.Lorem.characters(%Range{first: 1, last: 10})` but it give me back something with single quote and I couldn't use it directly into some Ecto Schema data for some reason. It was complaining about the data type. I am not sure what it was. So that's why I keep using my function

**Comment by tobyhinloopen** [comment]
@yordis `.characters` returns a list of characters, not a string (binary). Think of it as the difference between `[0x61, 0x62, 0x63]` and `"abc"` in Javascript. There are functions to convert them one-way-or-the-other.

**Comment by yordis** [comment]
@igas @tobyhinloopen any updates on this?

**Comment by tobyhinloopen** [comment]
Yes, there has been a `Faker.Util` module merged. See #85

**Comment by yordis** [comment]
@tobyhinloopen but is there anyway to just give me back random string without specify a format? My use case that become tedious because I just need a random string with specific length

**Comment by tobyhinloopen** [comment]
@yordis Why not use something like `:crypto.strong_rand_bytes(length)`?

**Comment by yordis** [comment]
@tobyhinloopen that's why I am using, but every single package that I use `faker` I have to add that function. The idea is that `faker` does those type of work 😢

**Comment by tobyhinloopen** [comment]
Do you look at `Faker.Util.join`?

EG `Faker.Util.join 3, &Faker.Util. lower_letter`

**Comment by yordis** [comment]
Forget about it! I will keep adding the function. I am looking conveniency, the function I posted you dont care about any formatting or any length (because there is a default value in my case) . Your alternative I have to keep using the formatting everywhere .... but don't worry, no a big deal, I thought it wouldn't hurt when somebody have the usecase

**Comment by tobyhinloopen** [comment]
Ah I get it now.

You're still free to create a PR but whatever 👍

**Comment by yordis** [comment]
@tobyhinloopen I can make the PR, which module would you use?

**Comment by tobyhinloopen** [comment]
Eh whichever you feel like? I think `Faker.Util` or something like `Faker.String`

**Comment by yordis** [comment]
@tobyhinloopen do you know when will be the next release? I am still needing this 😄

**Comment by tobyhinloopen** [comment]
@igas

**Comment by igas** [comment]
@tobyhinloopen it's been released in `0.9.0`.
